### ISSUE TITLE
Müşteri Hesap Kartında  "toplam borç", "toplam  alacak" "toplam  bakiye" 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ahmet328/sambapos#38]
```
Müşteri Hesap Kartında her satırda  ayrıntılı bilgi var ancak "toplam 
borç", "toplam  alacak" ve "toplam  bakiye" göremiyoruz.
```

Original issue reported on code.google.com by `isaakkay...@gmail.com` on 23 Sep 2011 at 11:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `emree...@gmail.com` on 24 Sep 2011 at 1:35
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `emree...@gmail.com` on 24 Sep 2011 at 3:03
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `emree...@gmail.com` on 29 Sep 2011 at 11:34
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5677c3e113f3.
```

Original comment by `emree...@gmail.com` on 29 Sep 2011 at 3:13
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Kontrol edildi,
```

Original comment by `mehmetak...@gmail.com` on 4 Oct 2011 at 12:01
- Changed state: **Verified**
